lib/OpenQA/Schema/ResultSet/Jobs.pm

```diff
@@ -225,10 +229,31 @@ sub create_from_settings {
     return $job;
 }
 
+sub search_modules ($self, $module_re) {
```

Comment:
Any chance we can re-use [the existing function](https://github.com/os-autoinst/openQA/blob/84e973fa8af13a159ecb68fad82bf0f250aeb34b/lib/OpenQA/WebAPI/Controller/API/V1/Search.pm#L73) rather than copying it here?

Reply:
Or split out a part of it for the sake of the argument

Reply:
isnt same , different command and different results needed

Reply:
I guess you're passing a regex rather than keywords. Besides that the grep is the same.